content/en/docs/File Format/implementationstatus.md

```diff
@@ -45,66 +45,66 @@ Implementations:
+| BYTE_STREAM_SPLIT                         |  ✅   |  ✅   |       |  ✅   |  ✅   |
 
 ### Compressions
 
 | Compression                               | C++   | Java  | Go    | Rust  | cuDF  |
 | ----------------------------------------- | ----- | ----- | ----- | ----- | ----- |
-| UNCOMPRESSED                              |  ✅   |  ✅   |       |       |  ✅   |
-| BROTLI                                    |  ✅   |  ✅   |       |       |  (R)  |
-| GZIP                                      |  ✅   |  ✅   |       |       |  (R)  |
-| LZ4 (deprecated)                          |  ✅   |  ❌   |       |       |  ❌   |
-| LZ4_RAW                                   |  ✅   |  ✅   |       |       |  ✅   |
-| LZO                                       |  ❌   |  ❌   |       |       |  ❌   |
-| SNAPPY                                    |  ✅   |  ✅   |       |       |  ✅   |
-| ZSTD                                      |  ✅   |  ✅   |       |       |  ✅   |
+| UNCOMPRESSED                              |  ✅   |  ✅   |       |  ✅   |  ✅   |
+| BROTLI                                    |  ✅   |  ✅   |       |  ✅   |  (R)  |
+| GZIP                                      |  ✅   |  ✅   |       |  ✅   |  (R)  |
+| LZ4 (deprecated)                          |  ✅   |  ❌   |       |  ✅   |  ❌   |
+| LZ4_RAW                                   |  ✅   |  ✅   |       |  ✅   |  ✅   |
+| LZO                                       |  ❌   |  ❌   |       |  ❌   |  ❌   |
+| SNAPPY                                    |  ✅   |  ✅   |       |  ✅   |  ✅   |
+| ZSTD                                      |  ✅   |  ✅   |       |  ✅   |  ✅   |
 
 ### Other format level features
 
 |                                           | C++   | Java  | Go    | Rust  | cuDF  |
 | ----------------------------------------- | ----- | ----- | ----- | ----- | ----- |
-| xxHash-based bloom filters                |  (R)  |  ✅   |       |       |  (R)  |
-| Bloom filter length (1)                   |  (R)  |  ✅   |       |       |  (R)  |
-| Statistics min_value, max_value           |  ✅   |  ✅   |       |       |  ✅   |
-| Page index                                |  ✅   |  ✅   |       |       |  ✅   |
-| Page CRC32 checksum                       |  ✅   |  ✅   |       |       |  ❌   |
-| Modular encryption                        |  ✅   |  ✅   |       |       |  ❌   |
-| Size statistics (2)                       |  ✅   |  ✅   |       |       |  ✅   |
+| xxHash-based bloom filters                |  (R)  |  ✅   |       |  ✅   |  (R)  |
+| Bloom filter length (1)                   |  (R)  |  ✅   |       |  ✅   |  (R)  |
+| Statistics min_value, max_value           |  ✅   |  ✅   |       |  ✅   |  ✅   |
+| Page index                                |  ✅   |  ✅   |       |  ✅   |  ✅   |
+| Page CRC32 checksum                       |  ✅   |  ✅   |       |  ✅   |  ❌   |
+| Modular encryption                        |  ✅   |  ✅   |       |  ❌   |  ❌   |
```

Comment:
Have to remember to update this cell when https://github.com/apache/arrow-rs/pull/6637 is merged.